src/main/java/com/google/sps/ServerMain.java

```diff
@@ -45,7 +81,38 @@ public static void main(String[] args) throws Exception {
     server.start();
     System.out.println("Server started!");
 
+    // https://stackoverflow.com/a/32228421
+    // Start thread to periodically delete old events
+    ScheduledExecutorService ses = Executors.newSingleThreadScheduledExecutor();
+    ses.scheduleAtFixedRate(new Runnable() {
+      @Override
+      public void run() {
+          deleteOldEvents();
+      }
+    }, 0, 1, TimeUnit.HOURS);
```

Comment:
Out of curiosity: How did you come up with the idea to fire the thread every hour?

Reply:
I was thinking about it and thought that deleting events every 24 hours in a specific time zone would be kind of strange for users in different time zones. By deleting roughly every hour, users anywhere will experience more or less the same thing.

Reply:
Sounds good! I was just curious.